.env.template

```
@@ -31,18 +43,10 @@ PRODUCER_STREAM=destination.subject
 # DESTINATION_URL=default
 # PRODUCER_STREAM=default
 
-# Uncomment this for Google Big Query Destination
-# DESTINATION_TYPE=bigQuery
-# GOOGLE_APPLICATION_CREDENTIALS="serviceAccount.json"
-# DESTINATION_URL=default
-# PRODUCER_STREAM=default
-# DATASET_ID=
-# TABLE_ID=
-
 APM_ACTIVE=false
 APM_SERVICE_NAME=relay-service
 APM_URL=http://localhost:8200/
 APM_SECRET_TOKEN=
 
 LOGSTASH_LEVEL='info'
-SIDECAR_HOST=0.0.0.0:15000
\ No newline at end of file
+SIDECAR_HOST=0.0.0.0:15000 # Comment for dev environment
```

```suggestion
SIDECAR_HOST=0.0.0.0:15000
```
This may create some confusion. The sidecar can run in dev environments and there are use-cases for it